### PR TITLE
ansible: add playbook for scheduling provisioning and test in cloud

### PR DESCRIPTION
**Pull request by rvykydal**
Create runner instances in cloud, run test, sync results, destroy runners.

### CODE REVIEW

File: linchpin/roles/schedule/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+
+- set_fact:
+    run_script_path: "{{ run_script_dir_path }}/{{ run_script_name }}"
+    service_file_path: "{{ systemd_dir_path }}/{{ service_name }}.service"
+    timer_file_path: "{{ systemd_dir_path }}/{{ service_name }}.timer"
```
Comment (jkonecny12) [inline linchpin/roles/schedule/tasks/main.yml]: Please change this to vars if possible.

File: linchpin/roles/schedule/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    src: kstests-cloud.timer.j2
+    dest: "{{ timer_file_path }}"
+    mode: 0664
+  tags: schedule
+
+- name: Restart the timer
+  systemd:
+    name: "{{ service_name }}.timer"
+    user: yes
+    state: restarted
```
Comment (jkonecny12) [inline linchpin/roles/schedule/tasks/main.yml]: Don't you also want to enable the timer?

File: linchpin/roles/schedule/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    path: "{{ timer_file_path }}"
+  tags: clean
+
+- name: Remove the run script {{ run_script_path }}
+  file:
+    state: absent
+    path: "{{ run_script_path }}"
+  tags: clean
+
+
```
Comment (jkonecny12) [inline linchpin/roles/schedule/tasks/main.yml]: Please remove two blank lines on the bottom of the file.

Comment (rvykydal) [reply linchpin/roles/schedule/tasks/main.yml]: Donw

File: linchpin/roles/schedule/tasks/main.yml (hunk 1 above)
Comment (rvykydal) [reply linchpin/roles/schedule/tasks/main.yml]: Done

File: linchpin/roles/schedule/tasks/main.yml (hunk 2 above)
Comment (rvykydal) [reply linchpin/roles/schedule/tasks/main.yml]: Perhaps yes, done

**Comment by rvykydal** [comment]
Applied the suggestions, also using two playbooks instead of tags now.

**Pull request merged**